common/scripts/update-imodeljs-native-version.sh

```diff
@@ -53,7 +53,16 @@ PbxProj2="$RepoRoot/test-apps/display-test-app/ios/imodeljs-test-app/imodeljs-te
 
 for PbxProj in $PbxProj1 $PbxProj2
 do
-  sed -i "s/version = .*;/version = $AddonVersion;/" "$PbxProj"
+  # Note: the '' seems to be required on MacOS to get around a strange "undefined label" error
+  sed -i '' "s/version = .*;/version = $AddonVersion;/" "$PbxProj"
+done
+
+# Update Android projects.
+BuildGradle1="$RepoRoot/test-apps/display-test-app/android/imodeljs-test-app/app/build.gradle"
```

Comment:
Do you anticipate more than 1 Android project in future?

Reply:
We could possibly create an Android core-test-runner in the future but that hasn't been discussed.

Reply:
Ok, it seems weird to have a loop for only one.